src/main/java/eu/darkbot/captcha/CaptchaSolver.java

```diff
@@ -49,6 +50,7 @@ public Map<String, String> solveCaptcha(URL url, String frontPage) {
 
         System.out.println("Captcha detected, opening solver...");
 
+        isFirstLogin = Main.API == null;
```

Comment:
This relies on a very obscure thing, that API isn't set during the initial login is probably not something that should be relied upon as it's subject to change.

The only reason first login is even a thing is because of another bot-specific behavior which should be changed, so i think this may do more harm than good in the long run